packages/template-chakra-storefront/src/components/product-view/index.jsx

```diff
@@ -574,7 +574,9 @@ const ProductView = forwardRef(
                                             // This is useful for mobile devices, this allows the user to pop open the keyboard and set the
                                             // new quantity with one click. NOTE: This is something that can be refactored into the parent
                                             // component, potentially as a prop called `selectInputOnFocus`.
-                                            e.target.select()
+                                            if (e.target.select) {
+                                                e.target.select()
+                                            }
```

Comment:
What was the reason that we had to add this guard? Its good to have, but I'm curious why before we didn't need it, and now we do.

Reply:
NumberInput from Chakra has bunch of other child components. And it looks like the select function is applicable only for the input field